app/controllers/audits_controller.rb

```diff
@@ -12,6 +12,13 @@ def show
 
   def save
     audit.user = current_user
+    # Get the next_item before updating the audit. As the
```

Comment:
Can you please add a test to drive this fix?

Can you think of a way to get rid of the comment via:
1. Writing more expressive code
2. Describing what you try to fix in the test?

Regardless the fix, it feels to me that it assumes that we rely on having the next item calculation precalculated, but as you described in https://github.com/alphagov/content-performance-manager/pull/198#issuecomment-311403441, it has many drawbacks.

So, it might be better to find a good solution for navigation rather than adding quick fixes?